glances/outputs/glances_unicode.py

```diff
@@ -15,11 +15,10 @@
     'ARROW_DOWN': ['\u2193', 'v'],
     'CHECK': ['\u2713', ''],
     'PROCESS_SELECTOR': ['>', '>'],
-    'MEDIUM_LINE': ['\u23af', '-'],
+    'MEDIUM_LINE': ['\u2500', '─'],
     'LOW_LINE': ['\u2581', '_'],
 }
 
-
```

Comment:
Ruff would be naggy with this missing empty line.

Reply:
Thanks! Will do 🤍